python/aitemplate/backend/rocm/conv2d/conv2d_bias_add.py

```diff
@@ -0,0 +1,92 @@
+#  Copyright (c) Meta Platforms, Inc. and affiliates.
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+#
+"""
+conv2d bias add codegen
+"""
+from ... import registry
+from . import common
+
+# pylint: disable=C0103,C0415,W0613,C0301
+
+
+@registry.reg("rocm.conv2d_bias_add_identity.config")
```

Comment:
Just being curious - why do we need to add ```conv2d_bias_add_identity```? I think the identity op would be eliminated by ```remove_id_ops``` pass, so we could only require ```conv2d_bias_add```. Or perhaps I misunderstood the meaning of "identity" here. Thanks!

Reply:
Yes, it's conv2d_bias_add but registry name called conv2d_bias_add_identity

Reply:
I see. Thanks for the clarification.